lib/utils/constants.dart

```diff
@@ -122,41 +70,118 @@ const kMaxRoyalty = 99.99;
 const kFileSizeLimitInGB = 32;
 const kMaxPriceLength = 14;
 const kSecInMillis = 1000;
+const double TABLET_MIN_WIDTH = 600;
+
+/// ````Reserved words, symbols, IDs etc
+const kCookbookId = 'cookbook_id';
+const kUsername = 'username';
 
-const kPylonSymbol = "upylon";
-const kUsdSymbol = "ustripeusd";
-const kAtomSymbol = "uatom";
-const kEuroSymbol = "eeur";
-const kAgoricSymbol = "urun";
-const kJunoSymbol = "ujunox";
-const kLunaSymbol = "uluna";
-const kUSTSymbol = "uusd";
+const kPylonSymbol = 'upylon';
+const kUsdSymbol = 'ustripeusd';
+const kAtomSymbol = 'uatom';
+const kEuroSymbol = 'eeur';
+const kAgoricSymbol = 'urun';
+const kJunoSymbol = 'ujunox';
+const kLunaSymbol = 'uluna';
```

Comment:
We are removing LUNA AND UST.

Reply:
@mustafapylons , removed all appearances of luna and ust